src/libraries/MessagesLib.sol

```diff
@@ -13,70 +13,69 @@ library MessagesLib {
     enum Call {
         /// 0 - An invalid message
         Invalid,
-        /// 1 - Add an asset id -> EVM address mapping
+        // --- Gateway ---
+        /// 1 - Proof
+        MessageProof,
+        /// 2 - Initiate Message Recovery
+        InitiateMessageRecovery,
+        /// 3 - Dispute Message Recovery
+        DisputeMessageRecovery,
+        /// 4 - Batch Messages
+        Batch,
+        // --- Root ---
+        /// 5 - Schedule an upgrade contract to be granted admin rights
+        ScheduleUpgrade,
+        /// 6 - Cancel a previously scheduled upgrade
+        CancelUpgrade,
+        /// 7 - Recover tokens sent to the wrong contract
+        RecoverTokens,
+        // --- Gas service ---
+        /// 8 - Update Centrifuge Gas Price
+        UpdateCentrifugeGasPrice,
+        // --- Pool Manager ---
+        /// 9 - Add an asset id -> EVM address mapping
         AddAsset,
-        /// 2 - Add Pool
+        /// 10 - Add Pool
         AddPool,
-        /// 3 - Allow an asset to be used as an asset for investing in pools
-        AllowAsset,
-        /// 4 - Add a Pool's Tranche Token
+        /// 11 - Add a Pool's Tranche Token
         AddTranche,
-        /// 5 - Update the price of a Tranche Token
+        /// 12 - Allow an asset to be used as an asset for investing in pools
+        AllowAsset,
+        /// 13 - Disallow an asset to be used as an asset for investing in pools
+        DisallowAsset,
+        /// 14 - Update the price of a Tranche Token
         UpdateTrancheTokenPrice,
-        /// 6 - Update the member list of a tranche token with a new member
-        UpdateMember,
-        /// 7 - A transfer of assets
+        /// 15 - Update tranche token metadata
+        UpdateTrancheTokenMetadata,
+        /// 16 - A transfer of assets
         Transfer,
-        /// 8 - A transfer of tranche tokens
+        /// 17 - A transfer of tranche tokens
         TransferTrancheTokens,
-        /// 9 - Increase an investment order by a given amount
+        /// 18 - Update the member list of a tranche token with a new member
+        UpdateMember,
+        /// 19 - Freeze tranche tokens
+        Freeze,
+        /// 20 - Unfreeze tranche tokens
+        Unfreeze,
+        /// --- Investment Manager ---
+        /// 21 - Increase an investment order by a given amount
         IncreaseInvestOrder,
```

Comment:
```suggestion
        RequestDeposit,
```

Reply:
True, good catch, thanks!

Reply:
@wischli changed to `DepositRequest` (and likewise for `RedeemRequest`) to be more consistent with the other messages